src/pages/blog/index.it.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+
+import Layout from '../../components/Layout'
+import BlogRoll from '../../components/BlogRoll_it'
```

Comment:
i import the BlogRoll italian version, in this way i filter the posts with the same language...